ajax/libs/color-scheme/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "color-scheme",
+  "filename": "color-scheme.min.js",
+  "version": "1.0.0",
+  "description": "Generate pleasant color schemes",
+  "main": "lib/color-scheme.js",
+  "scripts": {
+    "test": "mocha --compilers coffee:coffee-script/register",
+    "prepublish": "cake build"
+  },
```

Comment:
We don't need the `scripts` field. Please remove this field and all its contents.